src/api.ts

```diff
@@ -0,0 +1,8 @@
+const GOODS_URL = 'https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json';
+
+type GoodsLoader = () => Promise<Good[]>;
+
+export const getGoods: GoodsLoader = () => {
```

Comment:
это ты описываешь пропсы, а нужно описать то что будет отдавать
```suggestion
export const getGoods = (): GoodsLoader => {
```

Reply:
Все було ок, Діма помилився